app/v1/models/errors/mtdErrors.scala

```diff
@@ -33,6 +33,11 @@ object VrnFormatError extends MtdError(
   message = "The format of the supplied VRN field is not valid"
 )
 
+object PeriodKeyFormatError extends MtdError(
+  code = "FORMAT_PERIOD_KEY",
+  message = "The format of the supplied period key is not valid. It must be a 4 character string"
```

Comment:
This error message probably needs to be updated if we're planning on using a stricter regex for period key